types/json-schema/index.d.ts

```diff
@@ -642,3 +642,33 @@ export interface JSONSchema7 {
     writeOnly?: boolean;
     examples?: JSONSchema7Type;
 }
+
+export interface ValidationResult {
+    valid: boolean;
+    errors: ValidationError[];
+}
+
+export interface ValidationError {
+    property: string;
+    message: string;
+}
+
+/*
```

Comment:
Just FYI, these should use `**` so that they show up in quick info/signature help

Reply:
thx